### PR TITLE
Adding jsonpatch

### PR DESCRIPTION
**Pull request by brainstorm**
Breaking up dependencies for PR #3421 (OpenStack+Ansible).

### CODE REVIEW

File: recipes/jsonpatch/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{% set name = "jsonpatch" %}
+{% set version = "1.16 "%}
```
Comment (isuruf) [inline recipes/jsonpatch/meta.yaml]: Remove extra space after `1.16`

File: recipes/jsonpatch/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    - jsonpointer
+    - six
+
+test:
+  imports:
+    - jsonpatch 
+
+about:
+  home: https://github.com/stefankoegl/python-json-patch
+  license: BSD
```
Comment (isuruf) [inline recipes/jsonpatch/meta.yaml]: Add `license_file: COPYING` here

File: recipes/jsonpatch/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
```
Comment (synapticarbors) [inline recipes/jsonpatch/meta.yaml]: I believe only `python` and `setuptools` are required in the `build` section

File: recipes/jsonpatch/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+test:
+  imports:
+    - jsonpatch 
+
+about:
+  home: https://github.com/stefankoegl/python-json-patch
+  license: BSD
+  license_file: COPYING
+  summary: Applying JSON Patches in Python 2.6+ and 3.x
```
Comment (synapticarbors) [inline recipes/jsonpatch/meta.yaml]: Please add `dev_url` and `doc_url` links.

File: recipes/jsonpatch/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+    - jsonpointer
+    - six
+
+test:
+  imports:
+    - jsonpatch 
+
+about:
+  home: https://github.com/stefankoegl/python-json-patch
+  license: BSD
```
Comment (synapticarbors) [inline recipes/jsonpatch/meta.yaml]: Change to `BSD-3-Clause` to conform to https://opensource.org/licenses/alphabetical

File: recipes/jsonpatch/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+    - jsonpointer
+    - setuptools
+    - six
+
+  run:
+    - python
+    - jsonpointer
+    - six
+
+test:
```
Comment (synapticarbors) [inline recipes/jsonpatch/meta.yaml]: Please also test the command line scripts:

https://conda.io/docs/building/meta-yaml.html#test-commands

**Comment by isuruf** [comment]
jsonpatch is already available at https://github.com/conda-forge/jsonpatch-feedstock

**Comment by jakirkham** [comment]
Thanks @isuruf.

Also would encourage you, @brainstorm, to add yourself as a maintainer in the feedstock if you are interested. 😉

**Pull request closed without merging**